source/models/prediction.py

```diff
@@ -64,29 +69,36 @@ def prediction(user_input, model_encoder_dictionary):
     Args:
         user_input (list): user input edited into a 4 item list.
         model_encoder_dictionary (dict): dictionary with the model and encoder objects.
-
     """
-    class_value = model_encoder_dictionary["model"].predict(np.reshape(user_input,(1,4)))
-    print(class_value, '=', model_encoder_dictionary['encoder_mappings'][class_value[0]])
+    class_value = model_encoder_dictionary["model"].predict(
+        np.reshape(user_input, (1, len(user_input)))
+    )
```

Comment:
If you are expecting a single value, you could also simply get it immediately by adding `[0]`, so that `class_value` is the expected element, and not a list or array.